enginetest/engine_only_test.go

```diff
@@ -650,15 +650,33 @@ func TestTableFunctions(t *testing.T) {
 			Query:    "SELECT * from (select * from simple_TABLE_function(123)) as tf;",
 			Expected: []sql.Row{{"foo", 123}},
 		},
+		{
```

Comment:
Weren't there plan tests in the original PR as well? What happened to those?

Reply:
i don't like having disparate plan tests that have to be updated by hand. i can repro all of my errors with correctness tests so i'll stick with those for now.